app/Http/Services/GoogleService.php

```diff
@@ -4,11 +4,21 @@
 
 use Google\Cloud\Translate\TranslateClient;
 use Stichoza\GoogleTranslate\GoogleTranslate;
+use App\Http\Contracts\Services\GoogleServices as Contract;
 
-class GoogleService
+class GoogleService implements Contract
 {
     protected $clientTranslate;
```

Comment:
* [ ] Class property type annotations in doc blocks (for PHP < 7.4)

For class properties, we require the PHPDoc `@var` annotation for the datatype as class props prior to PHP 7.4 cannot be annotated natively